keyboards/kudox_game/rev1/config.h

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright 2019 Kumao Kobo <kumaokobo@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x9696
+#define DEVICE_VER      0x0100
+#define MANUFACTURER    Kumao Kobo
+#define PRODUCT         The Kudox Game Keyboard
+#define DESCRIPTION     Custom keyboard for game
+
+/* key matrix size */
+// Rows are doubled-up
+#define MATRIX_ROWS 15
+#define MATRIX_COLS 7
+
+// wiring of each half
+#define MATRIX_ROW_PINS { D4, D7, E6, B4, B5 }
+#define MATRIX_COL_PINS { F5, F6, F7, B1, B3, B2, B6 }
+// #define MATRIX_COL_PINS { B6, B2, B3, B1, F7, F6, F5} //uncomment this line and comment line above if you need to reverse left-to-right key order
+
+/* COL2ROW or ROW2COL */
+#define DIODE_DIRECTION COL2ROW
+
+/* define if matrix has ghost */
+//#define MATRIX_HAS_GHOST
+
+/* number of backlight levels */
+// #define BACKLIGHT_LEVELS 3
+
+/* Set 0 if debouncing isn't needed */
+#define DEBOUNCE 5
+
+/* serial.c configuration for split keyboard */
+#define SOFT_SERIAL_PIN D0
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+/* ws2812 RGB LED */
+#define RGB_DI_PIN D3
+
+#define RGBLED_NUM 14    // Number of LEDs
```

Comment:
This line is why your firmware isn't compiling.

The keyboard-level `config.h` file is processed before this file (which is revision-level). `keyboards/kudox_game/config.h` is defining `RGBLED_NUM` as 7, but then this line attempts to define it as 14.

```suggestion
#define RGBLED_NUM 7    // Number of LEDs
```

Also, personally I think all the RGB definitions should be here, but that matters less when there's only one revision being supported. But this line absolutely needs to be changed.